src/main/java/taxi/controller/driver/cars/GetMyCurrentCarsController.java

```diff
@@ -0,0 +1,26 @@
+package taxi.controller.driver.cars;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import taxi.lib.Injector;
+import taxi.model.Car;
+import taxi.service.CarService;
+
+public class GetMyCurrentCarsController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("taxi");
+    private final CarService carService = (CarService) injector
+            .getInstance(CarService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        List<Car> allByDriver = carService
+                .getAllByDriver((Long) req.getSession().getAttribute("driver_id"));
```

Comment:
Let's move argument of this method to a separate variable, to increase readability